### PR TITLE
Add airplane init tests

### PR DESCRIPTION
**Pull request by calufornia**
## Description

Adds tests for `airplane init` and variations. We add tests for:

`airplane tasks init`: JS, Python, SQL, REST, GraphQL, Shell, Docker, Workflow tasks, Non-inline tasks, Existing tasks, From runbooks
`airplane views init`: Regular view, Non-inline view
`airplane init`: Task -> task init, View -> view init

Resolves DTS-250

## Test plan
This entire PR

### CODE REVIEW

**Comment by calufornia** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1009** <a href="https://app.graphite.dev/github/pr/airplanedev/cli/1009" target="_blank">Graphite</a> 
    * **PR #1011** <a href="https://app.graphite.dev/github/pr/airplanedev/cli/1011" target="_blank">Graphite</a> 
      * **PR #1012** <a href="https://app.graphite.dev/github/pr/airplanedev/cli/1012" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/cli/1012?utm_source=stack-comment).

File: cmd/airplane/testutils/command.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	args []string,
+	fixtureDir string,
+) {
+	fixtureDir, err := filepath.Abs(fixtureDir)
+	require.NoError(err)
+
+	// Create a temporary directory and cd into it
+	tempDir, err := os.MkdirTemp("", "cli_test")
+	require.NoError(err)
+	//defer os.RemoveAll(tempDir)
```
Comment (leeweisberger) [inline cmd/airplane/testutils/command.go:40]: We should uncomment this so the temp dir is removed right?

File: cmd/airplane/testutils/command.go [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+		}
+
+		return defaultInclude(filePath, info)
+	}
+	CompareDirectories(require, fixtureDir, tempDir, equalWithPackageJSONMajorPinned, include)
+}
+
+// CompareDirectories compares the contents of two directories and returns true if all files in dir1 are in and equal
+// to a set of files in dir2.
+func CompareDirectories(
```
Comment (leeweisberger) [inline cmd/airplane/testutils/command.go:90]: lowercase this so it isn't exported

File: cmd/airplane/testutils/command.go [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+	pkg1 := build.PackageJSON{}
+	pkg2 := build.PackageJSON{}
+
+	err := json.Unmarshal(buf1, &pkg1)
+	require.NoError(err)
+
+	err = json.Unmarshal(buf2, &pkg2)
+	require.NoError(err)
+
+	compareDependencies(require, pkg1.Dependencies, pkg2.Dependencies)
```
Comment (leeweisberger) [inline cmd/airplane/testutils/command.go:155]: can we also compare the name that is generated?

**Review by leeweisberger — APPROVED** [review]
awesome stuff!

File: cmd/airplane/testutils/command.go (hunk 1 above)
Comment (calufornia) [reply cmd/airplane/testutils/command.go:40]: Oops good catch, I had commented this out for my own testing

**Pull request merged**